src/components/RestrictedDocumentSelection/RestrictedDocumentSelection.tsx

```diff
@@ -129,9 +129,11 @@ export const RestrictedDocumentSelection = trackComponent(
             suggestCountries={suggestCountries}
             handleCountrySelect={handleCountrySelect}
             placeholder={translate(
-              'doc_select.section.input_placeholder_country'
+              'doc_select.section.input_placeholder_country_copy' // lokalise is weird and input_placeholder_country_copy is `Select issuing country`
+            )}
+            noResults={translate(
+              'doc_select.section.input_placeholder_country' // // lokalise is weird and input_placeholder_country is `No Results`
             )}
-            noResults={translate('doc_select.section.input_country_not_found')}
```

Comment:
and keep this the same, and make sure the value is the same across other SDKs

Reply:
done